src/boundary/BoundaryController.h

```diff
@@ -7,52 +7,52 @@
 #ifndef ARTSS_BOUNDARY_BOUNDARYCONTROLLER_H_
 #define ARTSS_BOUNDARY_BOUNDARYCONTROLLER_H_
 
-
+#include <vector>
 #include "../field/Field.h"
-#include "Multigrid.h"
-#include "Surface.h"
-#include "Obstacle.h"
-#include "Boundary.h"
-#include "BoundaryData.h"
+#include "../utility/GlobalMacrosTypes.h"
 #include "../utility/tinyxml2.h"
+#include "../utility/Utility.h"
 #include "BoundaryDataController.h"
+#include "Multigrid.h"
+#include "Obstacle.h"
+#include "Surface.h"
 
 class BoundaryController {
-public:
+ public:
     static BoundaryController* getInstance();
     ~BoundaryController();
 
-    void applyBoundary(real *d, FieldType f, bool sync = true);
-    void applyBoundary(real *d, size_t level, FieldType f, bool sync = true);
-    //void applyBoundary(real *d, size_t level, FieldType f, real* val, bool sync = true); // for non-const BC
+    void apply_boundary(real *d, FieldType f, bool sync = true);
+    void apply_boundary(real *d, size_t level, FieldType f, bool sync = true);
+    // void apply_boundary(real *d, size_t level, FieldType f, real* val, bool sync = true); // for non-const BC
 
-    void printBoundaries();
-    void updateLists();
+    void print_boundaries();
+    void update_lists();
 
-    size_t getSize_innerList();
-    size_t getSize_boundaryList();
-    size_t* get_obstacleList();
-    size_t getSize_obstacleList();
+    size_t get_size_inner_list();
+    size_t get_size_boundary_list();
+    size_t* get_obstacle_list();
+    size_t get_size_obstacle_list();
 
-    size_t* get_innerList_level_joined();
-    size_t getSize_innerList_level_joined(); //TODO necessary?
-    size_t get_innerList_level_joined_start(size_t level);
-    size_t get_innerList_level_joined_end(size_t level);
+    size_t* get_inner_list_level_joined();
+    size_t get_size_inner_list_level_joined(); // TODO necessary?
+    size_t get_inner_list_level_joined_start(size_t level);
+    size_t get_inner_list_level_joined_end(size_t level);
 
-    size_t* get_boundaryList_level_joined();
-    size_t getSize_boundaryList_level_joined(); //TODO necessary?
-    size_t get_boundaryList_level_joined_start(size_t level);
-    size_t get_boundaryList_level_joined_end(size_t level);
+    size_t* get_boundary_list_level_joined();
+    size_t get_size_boundary_list_level_joined(); // TODO necessary?
+    size_t get_boundary_list_level_joined_start(size_t level);
+    size_t get_boundary_list_level_joined_end(size_t level);
 
-    size_t getSize_surfaceList() {return m_size_sList;};
+    size_t get_size_surfaceList() {return m_size_surface_list;};
 
-    size_t getObstacleStrideX(size_t id, size_t level);
-    size_t getObstacleStrideY(size_t id, size_t level);
-    size_t getObstacleStrideZ(size_t id, size_t level);
+    size_t get_obstacle_stride_x(size_t id, size_t level);
+    size_t get_obstacle_stride_y(size_t id, size_t level);
+    size_t get_obstacle_stride_z(size_t id, size_t level);
 
     std::vector<FieldType> get_used_fields();
```

Comment:
getters can be marked as const

Reply:
d56e0d5ac0a413b529789239ea126a66b109026f